### PR TITLE
Use the Fullname in top bar of Web interface and not the login

### PR DESCRIPTION
**Pull request by earendilfr**
To display the display name and not the login on Web interface.
Should be more readable sometime

### CODE REVIEW

File: lib/App/Netdisco/Web/AuthN.pm [hunk 1]
```diff
@@ -58,6 +58,7 @@ post '/login' => sub {
 
     if ($success) {
         session logged_in_user => param('username');
+        session logged_in_fullname => schema('netdisco')->resultset('User')->find(param('username'))->get_column('fullname');
```
Comment (ollyg) [inline lib/App/Netdisco/Web/AuthN.pm]: Good idea and thanks for the patch! I just want to see whether we can find a cleaner way of getting the fullname than a complete schema call, here, and then I will be happy to merge.

File: share/views/layouts/main.tt [hunk 2]
```diff
@@ -168,7 +168,7 @@
           [% ELSE %]
             <i class="icon-user"></i>
           [% END %]&nbsp;
-            [% session.logged_in_user | html_entity %] <b class="caret"></b>
+            [% session.logged_in_fullname | html_entity %] <b class="caret"></b>
```
Comment (ollyg) [inline share/views/layouts/main.tt]: I think we also want to fall back to username if the full name is not available (I believe full name is optional?).

Comment (earendilfr) [reply share/views/layouts/main.tt]: I will check if I can have a solution for this point :)

**Comment by ollyg** [comment]
Fixed and implemented - thanks!!

**Pull request closed without merging**